update(blog)!: Tails article with installation tutorial (#2869)
privacyguides/privacyguides.org: blog/posts/installing-and-using-tails.md

---
date:
    created: 2025-01-28T19:15:00
categories:
    - Tutorials
authors:
    - em
description: When browsing the web at home becomes dangerous to your safety, there are tools that can help minimizing your digital traces to stay safe. Tails is one of these tools. Here's why, when, and how you can install and use Tails.
schema_type: AnalysisNewsArticle
---

> # When the World Feels Dangerous, Stay Safe With Tails
> 
> ![Photo of a hand plugging a USB stick into a laptop and the Tails logo under it.](../assets/images/installing-and-using-tails/cover.webp)
> <small aria-hidden="true">Illustration: Jonah Aragon / Privacy Guides | Photo: Aleksander Dumała / Pexels</small>
> 
> There are many reasons someone might not feel safe to browse certain topics online. With all the information getting collected for each internet search, it is difficult to access sometime vital information without leaving a trace. These digital footprints might not threaten your personal safety if you are living with a supportive family, and in a democratic and free country. However, there are situations where someone might be put in great danger simply for looking at a website.<!-- more -->

Reviewer: ```suggestion
# When the World Feels Dangerous, Stay Safe With Tails

![Photo of a hand plugging a USB stick into a laptop and the Tails logo under it.](../assets/images/installing-and-using-tails/cover.webp)
<small aria-hidden="true">Illustration: Jonah Aragon / Privacy Guides | Photo: Aleksander Dumała / Pexels</small>

There is a growing number of people who no longer feel safe in their own home or country. Whatever the reason, many people might not feel safe to browse certain topics online. With all the information getting collected for each internet search, it is difficult to access sometime vital information without leaving a trace. These digital footprints might not threaten your personal safety if you are living with a supportive family, and in a democratic and free country. However, there are situations where someone might be put in great danger simply for looking at a website.<!-- more -->
```

Once you have reached out to a trusted ally in a secure way to request their help, and erased your browser's data for this site, do not consult this article again *if the digital traces of it might put you in danger*.

</div>

If you are completing this installation for someone else, or if the device your are currently using cannot put you at risk, here's why, when, and how you can install and use the portable system Tails:

## What Is Tails?

![Illustration of the Tails logo.](../assets/images/installing-and-using-tails/tails-logo-flat-inverted.svg)

<small aria-hidden="true">Illustration: Tails / Tor Project</small>

Tails is a portable *operating system* (a type of software like Windows and macOS) that is especially designed to minimize your digital footprints while using it.

The name comes from the acronym for "The Amnesic Incognito Live System". It is kept on a USB stick and resets itself entirely after each use (except if you enable its encrypted password-protected [Persistent Storage](https://tails.net/doc/persistent_storage/index.en.html)). What is done on Tails does not leave any digital traces on the computer it is plugged into.

> Additionally, Tails comes with pre-installed applications that will help increase your security and privacy online. When accessing the internet from Tails, your traffic will be automatically rerouted through the [Tor network](https://www.privacyguides.org/en/advanced/tor-overview/). This is a special network that makes it very difficult to identify your location or the websites you access, even from your Internet Service Provider (ISP). However, unless you configure the [Tor bridge](https://tails.net/doc/anonymous_internet/tor/index.en.html#hiding) option to hide this, your ISP will know you have been using Tor, but will not know which websites you have visited through Tor. It could have been anything. I personally use Tor when I have to visit Google Map, just to protect my data from Google's advertising.

Author: I'm okay with the rest of this suggestion but I would like to keep the sentence "It could have been anything." in it.

2. A trans or queer individual who lives with an unsupportive or hostile family and wishes to research trans or queer-related topics online, find communities, or access supportive resources without leaving any digital traces of their activities on a family device.

3. A democracy or human rights activist who organizes protests, communicate information online, or carry on any other activities that might have been declared unauthorized by an oppressive regime.

4. Any other situations where browsing the web or using a device anonymously might be necessary to protect someone's safety.

## When to Use Tails, and When Not to Use Tails

Tails will protect very well some information, but it will not magically protect everything. Before using it, read carefully what it can help you with and what it cannot do.

### Use Tails if you need to:

- Browse the web without leaving traces on your main computer.

- Use a computer without leaving traces of your activities on your main computer.

- Store information and process files in an encrypted way, away from your main computer.

- Hide which websites you visit from your ISP by using Tor, without leaving traces on your main computer.

> ### Be careful about:

Reviewer: ```suggestion
### Dangers when using Tails

Tails can't protect you from everything. You should still be careful and keep these points in mind:
```

- Your computer cannot securely boot from an external USB stick.

## How to Install and Use Tails

**Before you start, make sure that:**

1. The device you use for the installation is free from malware or spyware.

2. There is no recording software such as [Windows Recall](https://allthings.how/how-to-turn-off-windows-recall-ai-feature-in-windows-11-copilot-plus-pcs/) running. If there is, disable or pause it and delete your visit of this website from it.

3. You have a USB stick with a storage capacity of at least 8 GB. Ideally, I recommend using a fresh and new USB stick, but if this is not accessible to you, make sure you can erase this USB stick entirely and that the files on it were not sensitive or revealing information. Assume your USB stick could get seized later on and these deleted files could potentially get restored.

4. If you complete this installation for someone else, or if it is safe for you to do so (ordering online leaves a lot of digital footprints!), you may be interested in using a USB stick that looks more like a banal object. You can easily find cheap USB sticks on popular online stores that look like innocuous cartoon keychain charms, for example.

> **What you will need:**

Reviewer: ```suggestion
### What You'll Need
```

---
date:
    created: 2025-01-28T19:15:00
categories:
    - Tutorials
authors:
    - em
description: When browsing the web at home becomes dangerous to your safety, there are tools that can help minimizing your digital traces to stay safe. Tails is one of these tools. Here's why, when, and how you can install and use Tails.
schema_type: AnalysisNewsArticle
---

> # When the World Feels Dangerous, Stay Safe With Tails

Reviewer: ```suggestion
# Using Tails When Your World Doesn’t Feel Safe Anymore
```

There is a growing number of people who no longer feel safe in their own home or country. Whatever the reason, many people might not feel safe to browse certain topics online. With all the information getting collected for each internet search, it is difficult to access sometime vital information without leaving a trace. These digital footprints might not threaten your personal safety if you are living with a supportive family, and in a democratic and free country. However, there are situations where someone might be put in great danger simply for looking at a website.<!-- more -->

While this guide will be applicable to many, I am writing this article with these groups in mind:

1. Victims of domestic violence, 
2. Trans and queer individuals living in a hostile environment, and
3. Democracy and human rights activists located in regions adverse to their cause.

This article will help people in such situations learn how to browse the internet and use a computer in a more protected and anonymous way, in order to stay safe from harm.

<div class="admonition danger" markdown>

> <p class="admonition-title">A warning for those at very high risk</p>

Author: For consistency with the others: "A warning for those at very high risk" to "A Warning for Those at Very High Risk"

There are many good and legitimate reasons for using Tails. Here are a few examples from the scenarios I am considering in this article:

1. A victim of domestic violence who needs a secure way to research and communicate with shelters or other supportive resources to plan a safe escape, without leaving traces of their activities on a device accessible to the perpetrator.

2. A trans or queer individual who lives with an unsupportive or hostile family and wishes to research trans or queer-related topics online, find communities, or access supportive resources without leaving any digital traces of their activities on a family device.

3. A democracy or human rights activist who organizes protests, communicate information online, or carry on any other activities that might have been declared unauthorized by an oppressive regime.

4. Any other situations where browsing the web or using a device anonymously might be necessary to protect someone's safety.

## When to Use Tails, and When Not to Use Tails

Tails protects some data very well, but it will not magically protect everything. Before using it, read carefully what it can help you with and what it cannot do.

> ### Use Tails if you need to:

Reviewer: ```suggestion
### Good Uses for Tails

You should use Tails if you need to...
```

- If you share any files, be careful to **remove thoroughly any metadata** that could identify you from the file.

- If you share any pictures or videos, be extremely cautious with removing metadata and examining the picture or video to make sure no reflections or other details could inadvertently identify you.

- Do not to reuse any usernames, pseudonyms, email addresses, phone numbers, profile pictures, passwords, or any other information that you have used outside of Tails.

- Do not do anything that could identify you while using Tails. Assume that everything you do while using Tails could be linked together.

- Be careful with using any mobile data network to connect to the internet. Information related to [your mobile device could identify you](https://tails.net/doc/anonymous_internet/no-wifi/index.en.html).

- A very powerful adversary, such as a government, could potentially reidentify some information despite you using Tails. Read more about Tails' limitations here: [https://tails.net/doc/about/warnings/index.en.html](https://tails.net/doc/about/warnings/index.en.html)

> ### Do NOT use Tails if:

Reviewer: ```suggestion
### When to avoid Tails

You should consider **not** using Tails if...
```

Reviewer: ```suggestion
### When to avoid Tails

You should **not** use Tails if...
```

more firm
